src/dumbo/src/pdu/bytes.rs

```diff
@@ -131,7 +131,10 @@ pub trait NetworkBytesMut: NetworkBytes + DerefMut<Target = [u8]> {
     /// This method will panic if `offset` is invalid.
     #[inline]
     fn htonl_unchecked(&mut self, offset: usize, value: u32) {
-        (&mut self[offset..]).write_u32::<BigEndian>(value).unwrap()
+        let data = &mut self[offset..];
+        for (byte, read) in data.iter_mut().zip(u32::to_be_bytes(value).iter().cloned()) {
+            *byte = read as u8;
+        }
```

Comment:
Can we have a function for this as well ? Looks like `write_be_u32`. We can probably adjust the macros to take into consideration `be` vs `le`.